analyzers/tests/SonarAnalyzer.UnitTest/Extensions/SyntaxNodeExtensionsTest.cs

```diff
@@ -1232,5 +1211,19 @@ private static SyntaxNode NodeBetweenMarkers(string code, string language, bool
                 .GetCompilation()
                 .SyntaxTrees
                 .First();
+
+        private static ControlFlowGraph VerifyCfgCS<T>(string code) where T : CSharpSyntaxNode
```

Comment:
This isn't `Verify`, because it doesn't do the assertion.
```suggestion
        private static ControlFlowGraph CreateCfgCS<T>(string code) where T : CSharpSyntaxNode
```

Reply:
🤦

Reply:
In the TestHelper the options are to set the  `ignoreErrors` to true or false.
If I set it to false then even if I don't add the `// Error CSxxx` it does not fail which feels very wrong.
Being curious that in our UTs we just set the error comment and it works well -without specifying to ignore the compilation error - I found that in these tests, tt seems that the `SnipperCompiler`[ here](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SnippetCompiler.cs#L53) is pretty adamant on.